apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -28,6 +28,15 @@ const sortBy = ref(
 const choiceFilter = ref('any' as 'any' | 'for' | 'against' | 'abstain');
 const modalOpen = ref(false);
 const selectedVote = ref<Vote | null>(null);
+const containerWidth = ref(0);
+const el = ref<HTMLElement | null>(null);
+const { x } = useScroll(el, { behavior: 'smooth' });
+
+useResizeObserver(el, entries => {
```

Comment:
Can't you make it just with CSS?

Reply:
@bonustrack agreed that we can move on, it will be a bit slow if we run yarn dev but should be fine on prod
If someone is interested in solving I have created this https://jsbin.com/yihacif/edit?html,css,output